docs/source/process.rst

```diff
@@ -2,9 +2,8 @@ Data processing
 ===============
 
 .. attention::
-   Starting with version 0.5.0, the data processing interface ``Process``
-   is deprecated. Attempts to use ``Process`` will still
-   work at the moment but will no longer be available in the future.
+   The previous data processing interface ``Process`` was deprecated
+   starting from version 0.5.0.
```

Comment:
See my comment above in `convert.rst` about deprecation vs removal.

Reply:
removed!